header.php

```diff
@@ -18,7 +18,19 @@
 	<?php wp_head(); ?>
 </head>
 
-<body <?php body_class(); ?>>
+<body
+	<?php
+	if ( Go\AMP\is_amp() ) {
+		?>
+		aria-expanded="false"
+		[aria-expanded]="mainNavMenuExpanded ? 'true' : 'false'"
+		class="<?php echo esc_attr( implode( ' ', get_body_class() ) ); ?>"
+		[class]="'<?php echo esc_attr( implode( ' ', get_body_class() ) ); ?>' + ( mainNavMenuExpanded ? ' menu-is-open' : '' )"
```

Comment:
Minor point, but I've found for readability that this can be a good alternative for constructing `[class]` attributes:

```php
[class]="<?php echo esc_attr( sprintf( '%s + ( mainNavMenuExpanded ? " menu-is-open" : ""', wp_json_encode( implode( ' ', get_body_class() ) ) ) ); ?>"
```

Oh, something else: it turns out that `get_body_class()` is expensive. So really it should be called once and put in a variable, and then the variable can be used these two times in attributes.